### PR TITLE
Preprod: movie: Dune (2021) initial review + movie: Dune (1984) updates

### PR DESCRIPTION
**Pull request by paul-sheridan**
A 156 minute yawn fest. That said, an incestuous, desert love scene between Paul Atradies and Lady Jessica would have made it all worthwhile. Alas, it was not to be. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-1980s.st.txt [hunk 1]
```diff
@@ -5360,6 +5360,7 @@ romantic love [Paul and Chani.]
 uncle and nephew [Baron Harkonnen and Glossu Rabban.]
 mother and daughter [Lade Jessica and Alia.]
 what if I had to fight to the death [Feyd accepted Paul's challenge to a knife fight to the death and lost.]
+force field generator [People surrounded themselves in personal force fields for defense. Most notably, Paul and Duncan had a practice knife fight while enveloped in such protective energy fields.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1980s.st.txt:5363]: I think the theme should be simply "force field"

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -889,3 +889,51 @@ anti-gravity technology [The HVEC flying vehicle was said to have anti-gravity e
...
+Dune
+
+:: Description
+Dune (titled onscreen as Dune: Part One) is a 2021 American epic science
+fiction film directed by Denis Villeneuve and written by Villeneuve, Jon
+Spaihts, and Eric Roth. It is the first of a planned two-part adaptation of
+the 1965 novel by Frank Herbert, primarily covering the first half of the
+book.
+
+Synopsis: Set in the far future, it follows Paul Atreides as his family, the
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:905]: This is a break from the usual synopsis style

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 3]
```diff
@@ -889,3 +889,51 @@ anti-gravity technology [The HVEC flying vehicle was said to have anti-gravity e
...
+speculative spaceship [A fleet of Harkonnen vessels was shown over the planet Arrakis. A large, cigar-shaped vessel with a giant maw was shown over the planet Caladan.]
+force field generator [People surrounded themselves in personal force fields for defense. Most notably, Paul and Duncan had a practice knife fight while enveloped in such protective energy fields.]
+uncle and nephew [Glossu Rabba was the minion of his evil uncle, Baron Harkonnen.]
+coping with excruciating pain [Paul had to keep his hand inside of a pain box as part of a Bene Gesserit test.]
+clairvoyance [Paul inexplicably knew how to correctly don a "stillsuit" without ever having suited up in one before. The veteran Fremen ally Liet-Kynes was duly impressed with Paul as a result.]
+speculative environmental suit [The "stillsuits" recycled water and other waste products, enabling their wearers to survive in the desert for extended periods of time.]
+human familial relationship [Leto asked his consort, Lady Jessica, to protect their son, Paul, should anything happen to Leto.]
+mind control ability [Paul and Lady Jessica used a Bene Gesserit mind control technique on their captors.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:934]: Knock up? I think paul's abilities are central to the story, and my understanding is that he combines mind control with the ability to see into the future

Comment (odinlake) [reply notes/stories/film/film-scifi-2020s.st.txt:934]: It strikes me that he is some kind of saviour figure, like Jesus, as well - not sure if we have a theme for that..

**Review by odinlake — COMMENTED** [review]
Comments inline

File: notes/stories/film/film-scifi-1980s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1980s.st.txt:5363]: I couldn't agree more.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Good catch. I revised it.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Agree. Updated accordingly.

**Pull request merged**